config-model/src/test/cfg/significance/services.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
+<services>
+  <admin version="2.0">
+    <adminserver hostalias="node0"/>
+    <logserver hostalias="node0"/>
+  </admin>
+  <container version="1.0">
+    <search>
+        <significance>
+            <model language="en" model-id="idf-wiki-english" path="models/idf-english-wiki.json.zst"/>
+            <model language="no" path="models/idf-norwegian-wiki.json.zst" />
+            <model language="ru" url="https://some/uri/blob.json" />
+        </significance>
+    </search>
+  </container>
+  <content version="1.0">
+    <redundancy>1</redundancy>
+
+    <documents>
+      <document type="music" mode="index"/>
+    </documents>
+
+    <group name="mycluster">
+      <node hostalias="node0" distribution-key="0" capacity="0.79"/>
+    </group>
+  </content>
```

Comment:
Check whether this can be removed. Same for the `music.sd` file. The test should be as narrow as possible to reduce the cost of maintaining the test in the future.